src/core/edit/edit.model.js

```diff
@@ -10,5 +10,11 @@ export default class EditModel {
 		this.commit = new Command();
 		this.cancel = new Command();
 		this.reset = new Command();
+		this.commitShortcuts = {
+			'$default': 'tab|enter',
```

Comment:
With tabs and shift tabs everything is ok. ctrl+s causes saving of the html also as commiting of the text/number/password templates. But date template seems to not work with ctrl+s